conman/redirects/models.py

```diff
@@ -26,3 +26,10 @@ def save(self, *args, **kwargs):
         """Validate the Redirect before saving."""
         self.clean()
         return super().save(*args, **kwargs)
+
+
+class URLRedirect(Route):
+    """A `Route` that redirects to an arbitrary URL."""
+    handler = handlers.URLRedirectHandler.path()
+    target = models.URLField(max_length=2000)
+    permanent = models.BooleanField(default=False, blank=True)
```

Comment:
The `blank=True` isn't strictly necessary - [Django forces it in `BooleanField.__init__`](https://github.com/django/django/blob/1.10.4/django/db/models/fields/__init__.py#L977).